build/Jenkinsfile.branch

```diff
@@ -117,10 +117,16 @@ pipeline {
 
     stage ('Build Docker mattermost-enterprise-edition') {
       steps {
-        script {
-          platformStages.platformCheckoutDockerRepo('master')
-          def MMBINARYPATH="https://releases.mattermost.com/mattermost-platform/" + env.BRANCH_NAME + "/mattermost-enterprise-linux-amd64.tar.gz"
-          platformStages.platformBuildDockerImage(MMBINARYPATH, env.BRANCH_NAME + '.' + env.BUILD_NUMBER + '.' + SERVER_GIT_COMMIT)
+        dir('src/github.com/mattermost/mattermost-server') {
+          withCredentials([usernamePassword(credentialsId: 'materbuild-docker-hub', usernameVariable: 'DOCKER_USER', passwordVariable: 'DOCKER_PASS')]) {
```

Comment:
Is `materbuild-docker-hub` a typo?

Reply:
it is but the typo is in jenkins. will fix and update

Reply:
fixed